towhee/functional/__init__.py

```diff
@@ -41,6 +41,10 @@
 
 to_video = DataCollection.to_video
 
+load_video = DataCollection.load_video
```

Comment:
consider using `read_video` and `read_audio` for data source, and `to_video` and `to_audio` for data sink;

Reply:
Fixed